public/js/pimcore/object/classes/data/data.js

```diff
@@ -25,7 +25,7 @@ pimcore.object.classes.data.data = Class.create({
         "versions", "properties", "permissions", "permissionsforuser", "childamount", "apipluginbroker", "resource",
         "parentClass", "definition", "locked", "language", "omitmandatorycheck", "idpath", "object", "fieldname",
         "property", "localizedfields", "parentid", "scheduledtasks", "latestVersion", "haschildren", "siblings", "hassiblings",
-        "childrenSortby", "childrensortorder", "versioncount", "dirtylanguages", "dirtyfields", "classtitle"
+        "childrensortby", "childrensortorder", "versioncount", "dirtylanguages", "dirtyfields", "classtitle", "classid"
```

Comment:
Nice catch, LGTM but i've noticed that it breaks for every other getter as well eg. `valueForFieldName`, `dao`, `NextParentForInheritance` ,`ClosestParentOfClass`, `LazyLoadedFieldNames`

Could you please add these as well? TIA

![image](https://github.com/pimcore/admin-ui-classic-bundle/assets/6014195/6280a263-5448-48e3-9646-662274e394ec)
![image](https://github.com/pimcore/admin-ui-classic-bundle/assets/6014195/2d8c498f-04c6-4e28-9288-4fea997d7466)
![image](https://github.com/pimcore/admin-ui-classic-bundle/assets/6014195/ca3a2fe7-94ab-47db-ae21-7ccee8c9fc57)

Reply:
@kingjia90 Added it (in php also localizedfields was missing). I also sorted it alphabetically. So we see faster missing properties.

Reply:
Thank you for the changes and sorting, that's helpful.

Indeed, it's a little hard to read and manage such a long array, can't break a line for each element.
I am having a sort of brain-fart, what do you think about a "counter-intuitive" (just for human/dev readability) solution to make it an associative array, group by similar domain/areas and that gets flattened for the compiler? 🙈

Reply:
@kingjia90 I don't think I quite follow you. Do you mean to make a multidimensional list out of it?

Maby we generate the list via Ajax call? Via PHP reflection?

We can write it down as an issue. But I think it's too much for this PR.

Reply:
Yes, some kind of that, but agree, that's not worth it to invest so much time on that.

So far LGTM as is, i will have a deeper review and merge soon or latest on Monday. Wish you a nice weekend!

Reply:
@kingjia90 I wish you also a nice weekend!